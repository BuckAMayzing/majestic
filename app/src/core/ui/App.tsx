import * as React from 'react';
import { observer, Provider } from 'mobx-react';
import { ThemeProvider } from 'styled-components';
import Home from './Home';
import stores from './stores';
import theme from './theme';

import '@blueprintjs/core/lib/css/blueprint.css';
import '@blueprintjs/icons/lib/css/blueprint-icons.css';

const App = () => (
  <ThemeProvider theme={theme}>
    <Provider {...stores}>
      <Home
        workspace={stores.workspace}
        debug={stores.debug}
        tests={stores.tests}
        searcher={stores.searcher}
        preference={stores.preference}
        updater={stores.updater}
      />
    </Provider>
  </ThemeProvider>
);

export default observer(App);
