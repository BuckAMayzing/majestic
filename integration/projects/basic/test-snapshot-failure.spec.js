import renderer from "react-test-renderer";
import React from "react";
import App from "./app";

describe("test", () => {
  it("snapshot test", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
