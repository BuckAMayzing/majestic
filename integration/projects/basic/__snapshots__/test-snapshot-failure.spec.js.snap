// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot test 1`] = `
<div
  className="App"
>
  Hello world
</div>
`;
